js/interface.js

```diff
@@ -1,13 +1,16 @@
 var widgetId = Fliplet.Widget.getDefaultId();
 var data = Fliplet.Widget.getData() || {};
+var page = Fliplet.Widget.getPage();
+
+data.omitPages = page ? [page.id] : [];
 
 var linkActionProvider = Fliplet.Widget.open('com.fliplet.link', {
   // If provided, the iframe will be appended here,
   // otherwise will be displayed as a full-size iframe overlay
   selector: '#action',
   // Also send the data I have locally, so that
   // the interface gets repopulated with the same stuff
-  data: data.action,
+  data: data,
```

Comment:
Why is this changed? This looks like it'll affect how current widget instances load configurations.

Reply:
@tonytlwu  Fixed